capture-sdk/sdk/src/main/res/values/styles.xml

```diff
@@ -216,6 +217,7 @@
     <style name="Root.GiniCaptureTheme.Help.PhotoTips.Tip.TextStyle" parent="TextAppearance.AppCompat">
```

Comment:
[Clarification] This style is not used anymore, right? Just asking because I see you modified it.

Reply:
Yeah probably. I will check in the morning.